### PR TITLE
enrollments reset feature flag

### PR DESCRIPTION
**Pull request by zacharis278**
This feature flag will allow an endpoint to delete all enrollments to be included in the lms

Configuration Pull Request
---

Make sure that the following steps are done before merging:

  - [ ] A DevOps team member has approved the PR if it is code shared across multiple services and you don't own all of the services.
  - [x] Are you adding any new default values that need to be overridden when this change goes live? If so:
    - [ ] Update the appropriate internal repo (be sure to update for all our environments)
    - [ ] If you are updating a secure value rather than an internal one, file a DEVOPS ticket with details.
    - [ ] Add an entry to the CHANGELOG.
  - [ ] If you are making a complicated change, have you performed the proper testing specified on the [Ops Ansible Testing Checklist](https://openedx.atlassian.net/wiki/display/EdxOps/Ops+Ansible+Testing+Checklist)?  Adding a new variable does not require the full list (although testing on a sandbox is a great idea to ensure it links with your downstream code changes).
  - [ ] Think about how this change will affect Open edX operators.  Have you updated the wiki page for the next Open edX release?

### CODE REVIEW

File: playbooks/roles/edxapp/defaults/main.yml [hunk 1]
```diff
@@ -302,6 +302,7 @@ EDXAPP_FEATURES:
   ENABLE_READING_FROM_MULTIPLE_HISTORY_TABLES: "{{ EDXAPP_ENABLE_READING_FROM_MULTIPLE_HISTORY_TABLES }}"
   SHOW_HEADER_LANGUAGE_SELECTOR: "{{ EDXAPP_SHOW_HEADER_LANGUAGE_SELECTOR }}"
   SHOW_FOOTER_LANGUAGE_SELECTOR: "{{ EDXAPP_SHOW_FOOTER_LANGUAGE_SELECTOR }}"
+  ENABLE_ENROLLMENT_RESET: false
```
Comment (Zacharis278) [inline playbooks/roles/edxapp/defaults/main.yml]: if this only ever included in sandbox builds should this be omitted?

**Review by schenedx — APPROVED** [review]
👍

**Review by kdmccormick — COMMENTED** [review]
I think you'll also need to add `EDXAPP_ENABLE_ENROLLMENT_RESET` with default `false`.

**Comment by zacharis278** [comment]
@kdmccormick yeah I guess it makes sense to have the default here even if it's never directly referenced

**Pull request merged**